### ISSUE TITLE
No matching package version found for modules with letters in the version name

### ISSUE THREAD

**Issue by Xennis** [meanmail-dev/requirements#59]
**Describe the bug**
The plugin fails for the Python module [black](https://pypi.org/project/black/):

```
No matching package version found: 'black==20.8b1' (required: ==20.8b1, installed: 20.8b1, latest: <nothing>)
```

**To Reproduce**
1. Add `black==20.8b1` to a requirements.txt

**Expected behavior**
Plugin can handle Python modules that are properly released on PyPi: https://pypi.org/project/black/20.8b1/ Especially because black is with nearly [20.000 GitHub stars](https://github.com/psf/black) really popular.

**Plugin (please complete the following information. See Main Menu -> File -> Settings -> Plugins):**
 - Version 2021.3-203

**Comment by Xennis** [comment]
Dear @meanmail is there any update on this issue?

**Comment by Pytlicek** [comment]
+1 for this issue. Any updates?

**Comment by camUrban** [comment]
I'm also having this issue!

**Comment by meanmail** [comment]
Fixed in 2022.2

**Comment by Pytlicek** [comment]
Many thanks for sorting this out  🙏👍
